website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+page_title: CTS v0.5.0
+description: >-
+  Consul-Terraform-Sync Configuration Deprecations for v0.5.0
```

Comment:
```suggestion
  Consul-Terraform-Sync release notes for v0.5.0
```
I wonder if we should keep this more generic. Or is the intention for these pages to always be for deprecations?

Reply:
Good question. I'm not sure what the future of these pages will be! Right now, I was thinking it may be any miscellaneous release-specific documentation.

One hesitation I have about changing the title to "release notes" is that it only covers a subset of changes in the release. Maybe, naming the section "Release Notes" isn't a good idea. Let me know if you have thoughts

Reply:
I imagine in the future there might be other changes in these pages besides deprecation. In the future, others will look at this page as a starting point, so whatever we decided will be the pattern in the future.

Reply:
Makes sense. What are your thoughts on going with your suggestion of "Consul-Terraform-Sync Release Notes for v0.5.0" and then below add a sentence to clarify, e.g. "The release notes for v0.5.0 will focus on a subset of configuration deprecations."